db/queries/application/queries.py

```diff
@@ -178,6 +183,23 @@ def get_count_by_status(round_ids: Optional[list] = [], fund_ids: Optional[list]
     return results
 
 
+def create_qa_base64file(application_data: dict, with_questions_file):
+    if with_questions_file is not None and with_questions_file:
```

Comment:
is `with_questions_file` a boolean? Can you add typing to the method signature, and if it's a bool then you can just do `if with_questions_file:`. Even if it's an object `if with_questions_file:` will just resolved to `False` if it's `None` so can make this if block simpler

Reply:
Fixed